README.md

```diff
@@ -28,4 +28,4 @@ implement the ability to filter and sort people in the table.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_people-table-advanced/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://vanya-kalyenichenko.github.io/react_people-table-advanced/) and add it to the PR description.
```

Comment:
Add this link to the PR description
<img width="951" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/64750958/a23c9a19-4008-46f4-8781-2cf05d8fe765">